tests/Mail/MailMarkdownTest.php

```diff
@@ -23,7 +23,7 @@ public function testRenderFunctionReturnsHtml()
 
         $result = $markdown->render('view', []);
 
-        $this->assertTrue(strpos($result, '<html></html>') !== false);
+        $this->assertContains('<html></html>', (string) $result);
```

Comment:
I forced `string` here, [and line 42](https://github.com/laravel/framework/pull/22461/files#diff-9469ae15fe70b31a71f0aabdcf02d7f2R42), because assertContains expected and 2nd argument been an array, traversable or string.